server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -200,14 +202,14 @@ export function configureWebSocketServices(
                 isTokenExpiryEnabled);
             if (!claims) {
                 // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject("Invalid claims");
+                return Promise.reject({ code: 401, message: "Invalid claims" });
```

Comment:
I am guessing 401 is retryable and 403 is not. If so, this should be non retryable as well.

Reply:
`validateTokenClaims` includes a check for token expiry, so there's a chance this is retryable. Should I remove that check from `validateTokenClaims`? Or perhaps throw the error within `validateTokenClaims` for greater control? The only other place `validateTokenClaims` is used is within Alfred's [routes/api/api](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts), which has a similarly indistinguishable error being thrown.

Reply:
Perhaps, to maintain scope, I can create a follow-up PR that splits `validateTokenClaims` to not include token expiry.

Reply:
Sounds good to me.